erikbenedict/weather-outlook
# Feature/readme

## Test

- Go to repository page
- Read over for typos and grammatical errors
- Able to see screenshot of application in the 'Screenshots' section
- Click link in the 'Deployed Application' section
- Follow instructions from the 'Instructions' section
